openbook_common/helpers.py

```diff
@@ -142,3 +143,16 @@ def write_in_memory_file_to_disk(in_memory_file):
     tmp_file.seek(0)
     tmp_file.close()
     return tmp_file
+
+
+def send_alert_to_channel(alert):
+
+    post = {"text": alert}
+
+    json_data = dumps(post)
+    req = requests.post(ALERT_HOOK_URL,
+                       data=json_data.encode('utf-8'),
+                       headers={'Content-Type': 'application/json'})
+
+    if rq.status_code != requests.status_codes.codes.ok:
```

Comment:
shoudnt this be `req.status_code` ?

Reply:
Oops, I missed that. Will do that. Thanks!